package.json

```diff
@@ -60,6 +60,7 @@
     "node-fetch": "^2.6.0",
     "synthetix-js": "=2.26.2",
     "truffle": "^5.1.37",
-    "truffle-hdwallet-provider": "^1.0.0-web3one.1"
+    "truffle-hdwallet-provider": "^1.0.0-web3one.1",
+    "typescript-logging": "^1.0.0"
```

Comment:
This dependency is needed to run:
```
npx truffle exec ./node_modules/@gnosis.pm/dex-contracts/build/common/scripts/ganache/setup_thegraph_data.js
 ```
For this PR: https://github.com/gnosis/dex-subgraph/pull/69

one could argue that hence, this dependency should go into dex-contracts dependencies, but I guess we wanted to keep this package: dex-contracts, as small as possible.

@fleupold, what is your opinion? How strong is this argument to keep dex-contracts npm module small?